lib/puppet/util/puppetdb_validator.rb

```diff
@@ -10,7 +10,7 @@ class PuppetdbValidator
       attr_reader :test_path
       attr_reader :test_headers
 
-      def initialize(puppetdb_server, puppetdb_port, use_ssl=true, test_path = "/v3/version")
+      def initialize(puppetdb_server, puppetdb_port, use_ssl=true, test_path = "/pdb/query/v4/version")
```

Comment:
Okay, this makes the module defaulting to behaviour not yet released, so it will break stuff. We've threatened to do this for a while, but I think we need to coordinate this properly, we might need to cut a new set of branches to handle the new state as we start to make breaking changes (ie. stable/master etc.). Lets have a chat about this before merging, make sure we have a plan for module v5.0.